build/buildall.sh

```diff
@@ -14,6 +14,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+
+# NOTE Passing MPLAYER_NEW=1 when calling this script 
+# will cause the newer mplayer to be built
+export MPLAYER_NEW=1
```

Comment:
Can you change this to use ?= or := (I forget which one sets it only if not set already) so that then it can be overridden without editing the build file directly?